openupgrade_scripts/readme/ROADMAP.rst

```diff
@@ -0,0 +1,778 @@
+Module coverage 13.0 -> 14.0
```

Comment:
There is already https://github.com/OCA/OpenUpgrade/blob/14.0/doc/source/modules130-140.rst

Reply:
> There is already https://github.com/OCA/OpenUpgrade/blob/14.0/doc/source/modules130-140.rst. 

the idea is to have the master file in the readme folder of the script module. and then to have it also up to date in the documentation, because of the wget [here](https://github.com/OCA/OpenUpgrade/pull/2662/files#diff-491a891e27f244dd6c91aa9da9d99a0422b08ecfab3f25a09765a2b9180910d2R10) **as all the other coverage files** (of the other revision)

So, finally
- ``/doc/`` is for all the general documentation, which should not move much
- ``openupgrade_scripts`` is for all the scripts, and i consider this coverage file as the roadmap of this module, so when we propose a script, we change only files of a single folder.

> In the roadmap, we can link to this doc instead.

Indeed, if you don't like this move, we can just consider to add a link to the coverage section. It's better than nothing.

Reply:
I vote for the link option. It's not good to maintain duplicated .rst files.